src/styles.scss

```diff
@@ -21,15 +21,19 @@ body {
   font-weight: normal;
 }
 
-* {
+.bbmri-transition,
+.bbmri-transition a,
+.bbmri-transition div {
   -webkit-transition: all .2s ease-out;
   -moz-transition: all .2s ease-out;
   -o-transition: all .2s ease-out;
   -ms-transition: all .2s ease-out;
   transition: all .2s ease-out;
 }
 
-*:hover {
+.bbmri-transition:hover,
+.bbmri-transition a:hover,
+.bbmri-transition div:hover {
```

Comment:
You missed the standard and the "-ms" variant here.

Reply:
I do not see where I missed them? My code (lines 34 - 42) looks like:

.bbmri-transition:hover,
.bbmri-transition a:hover,
.bbmri-transition div:hover {
  -webkit-transition: all .2s ease-in;
  -moz-transition: all .2s ease-in;
  -o-transition: all .2s ease-in;
  -ms-transition: all .2s ease-in;
  transition: all .2s ease-in;
}

Reply:
ok was a problem with code folding.